content/docs/usage/setup/configuration.mdx

```diff
@@ -2,4 +2,55 @@
 title: Configuration
 ---
 
-Coming Soon.
+
+## Configuration
+
+After the installation, you'll need to go through the initial configuration,
+
+#### 1) Terms [#configuration-1]
+
+Welcome to Dogebox! Please read and accept the terms and conditions
+
+![](../../../../public/static/image/docs/configuration/configuration-terms.png)
+
+#### 2) Config [#configuration-2]
+
+Set your device name and preferred keyboard layout.
+Also select your mass storage disk, note this may be different to your installation location and if you plan to sync the Dogecoin blockchain will require plenty of space.
+
+![](../../../../public/static/image/docs/configuration/configuration-config.png)
+
+Advanced options allow you to disable the use of the Dogecoin Foundation's binary caches.
+These binary caches contain pre-build binaries for Dogebox OS and the Foundation's supported Pups, which when used will speed up the installation process.
+
+![](../../../../public/static/image/docs/configuration/configuration-config-advanced.png)
+
+#### 3) Password [#configuration-3]
+
+Set your password, make sure it's secure!
```

Comment:
Mention that it is the password used for the Web Admin interface, and not the ssh or commandline login user?

Reply:
also maybe mention this encrypts your root key and seed phrase - make sure it's secure?